pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -84,18 +84,25 @@ func newCephStatusChecker(context *clusterd.Context, clusterInfo *cephclient.Clu
 }
 
 // checkCephStatus periodically checks the health of the cluster
-func (c *cephStatusChecker) checkCephStatus(context context.Context) {
+func (c *cephStatusChecker) checkCephStatus(monitoringRoutines map[string]*opcontroller.ClusterHealth, daemon string) {
 	// check the status immediately before starting the loop
-	c.checkStatus(context)
+	c.checkStatus(monitoringRoutines[daemon].InternalCtx)
 
 	for {
+		// We must perform this check otherwise the case will check an index that not exist anymore and
```

Comment:
This same comment is in four places to update
```suggestion
		// We must perform this check otherwise the case will check an index that does not exist anymore and
```